sdk/rust/src/lib.rs

```diff
@@ -170,3 +172,6 @@ pub trait Plugin: Send + Sync + 'static {
 		})
 	}
 }
+
+// Re-export macros
+pub use hipcheck_sdk_macros::query;
```

Comment:
should this be added to the prelude import, so it is imported with `use hipcheck_sdk::prelude::*`

Reply:
good question. I think the answer is yes, but @alilleybrinker might have opinions on default-importing proc macros. lets say yes to including it for now

Reply:
That would make sense. I can move it.

Reply:
it could be put behind a macros feature flag and be imported if `macros` is enabled

Reply:
actually yeah I like that. michael can you implement that?

Reply:
Let me know if it looks good now.